meddlr/metrics/build.py

```diff
@@ -54,7 +56,16 @@
     "DSC": (DSC, {}),
     "VOE": (VOE, {}),
     "ASSD": (ASSD, {"connectivity": 1}),
-    "CV": (CV, {})
+    "CV": (CV, {}),
+
+    # ======== Feature Based Metrics ============= #
+    "LPIPS": (LPIPS, {"net_type": "vgg",
+                      "mode": "grayscale",
+                      "lpips": True,
+                      "pretrained": True
+                      }),
+    "SSFD": (SSFD, {"mode": "grayscale",
+                    "layer_names": ['block4_relu2']})
```

Comment:
Same here - if this is the default configuration in the SSFD paper, comment as such